geniza/urls.py

```diff
@@ -15,24 +15,31 @@
 """
 from django.conf import settings
 from django.conf.urls.i18n import i18n_patterns
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
+from django.urls.conf import re_path
 from django.views.generic.base import RedirectView
+from wagtail.admin import urls as wagtailadmin_urls
+from wagtail.core import urls as wagtail_urls
 
 urlpatterns = [
     # redirect homepage to admin site for now
-    path("", RedirectView.as_view(url="admin/")),
     path("admin/", admin.site.urls),
     path("accounts/", include("pucas.cas_urls")),
     path("i18n/", include("django.conf.urls.i18n")),
     path("taggit/", include("taggit_selectize.urls")),
+    path("cms/", include(wagtailadmin_urls)),
 ]
 
 # urls that should be available in multiple languages
 urlpatterns += i18n_patterns(
     path("", include("geniza.corpus.urls", namespace="corpus")),
+    path("", include(wagtail_urls)),
 )
 
+# Media URLs for wagtail
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
this should be debug only, for the real deploy we serve out via nginx 

(I don't think this is related to the visual test problem, just thought I would flag)